running locally/docker-prestissimo/prestissimo/catalog/jmx.properties

```diff
@@ -0,0 +1,5 @@
+connector.name=jmx
+
+jmx.dump-period=10s
+jmx.dump-tables=java.lang:type=Runtime,com.facebook.presto.execution.scheduler:name=NodeScheduler
+jmx.max-entries=86400
```

Comment:
please always have a newline at the bottom of a file

Reply:
Done